src/com/googlecode/shavenmaven/Dependencies.java

```diff
@@ -49,11 +49,11 @@ public boolean update(File directory) {
     }
 
     public static void main(String[] args) throws Exception {
-        if (args.length == 0 || args.length > 2) {
```

Comment:
Not so sure about this change, was there a specific reason to go this way or just cleanness?!